schematools/contrib/django/auth_backend.py

```diff
@@ -0,0 +1,95 @@
+from django.contrib.auth.backends import BaseBackend
+from schematools.contrib.django.models import (
+    Profile,
+    generate_permission_key,
+    split_permission_key
+)
+
+PERMISSION_MATRIX = [
```

Comment:
MATRIX suggereert 2 dimensies, het is toch eerder een LIST?

Reply:
Ouch, goede. Ik dacht om deze lijst combineren met mutaties matrix, maar mutaties blijven dan in DSO-API.